tests/everest/test_math_func.py

```diff
@@ -12,117 +12,12 @@
 from everest.suite import _EverestWorkflow
 from everest.util import makedirs_if_needed
 
-CONFIG_FILE_MINIMAL = "config_minimal.yml"
-CONFIG_FILE_IN_CONSTR = "config_in_constr.yml"
-CONFIG_FILE_OUT_CONSTR = "config_out_constr.yml"
```

Comment:
Could you check if these files are used in some other test, and if not, remove them?

Reply:
I think they are (only) used in the documentation of Everest as examples for slowly building up the complexity of using Everest (at least the minimal one is for sure, maybe the in_constr and out_constr aren't)

Reply:
But in the documentation are they explicitly referring to the file location? Because they used to be in an `examples `folder, but that has now become `test-data`, and we should not have anything there that is not used. The documentation should not refer to `test-data`.

Reply:
Sorry you are right, they are making their own ('everest_config.yml') which happens to look exactly the same as the 'config_minimal.yml'. We can delete them I guess, and also clean up the associated 'jobs' I suppose?

Reply:
yes, please do.

Reply:
The job (DISTANCE3) for in- and output constraint was used in other tests. Also, config_minimal was used in quite a lot of other tests too, so we need to keep the config file.